main/test/ca/uwaterloo/flix/library/LibrarySuite.scala

```diff
@@ -29,7 +29,8 @@ class LibrarySuite extends Suites(
   new TestInt64,
```

Comment:
The tests above (ie TestBigInt) seemed to fail when I run the LibrarySuite (I commented them out for running the Set unit tests).  I'm not sure why these tests fail.